app/views/react.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>React Stuff</title>
+    <link rel="stylesheet" type="text/css" href="/sweetalert.css">
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
Doesn't appear that this is getting used anywhere. If you come across a need for jQuery I don't mind adding it via NPM but not like this.